### ISSUE TITLE
Should be able to click on photos to get higher resolution

### ISSUE THREAD

**Issue by jonfroehlich** [jonfroehlich/makeabilitylabwebsite#692]
As now, photos are just small thumbnails--would be nice to view larger sizes

![image](https://user-images.githubusercontent.com/1621749/48291607-f3cfab00-e42b-11e8-9144-6fc002f2486c.png)

**Comment by jonfroehlich** [comment]
Duplicate of https://github.com/jonfroehlich/makeabilitylabwebsite/issues/676 but I'll close https://github.com/jonfroehlich/makeabilitylabwebsite/issues/676.

**Comment by imiao97** [comment]
That look good, what do you think about something like this? https://www.w3schools.com/howto/howto_css_zoom_hover.asp
where it zoom in when hovered and back to normal when unhovered.

**Comment by jonfroehlich** [comment]
I was thinking something more like what we do here: https://sidewalk-test.cs.washington.edu/labelingGuide

Specifically:
![image](https://user-images.githubusercontent.com/1621749/52000697-92d7d200-2472-11e9-9261-2e4ef830f4eb.png)

And then you click on a thumbnail and it enlarges like this:
![image](https://user-images.githubusercontent.com/1621749/52000723-a3884800-2472-11e9-9c52-ec8586c554b8.png)

Where we show a caption on the bottom. Then, on the left and right of the image we should have previous and advance arrows so users can stay in this zoomed in view and go back and forth between images on that page.

**Comment by jonfroehlich** [comment]
@imiao97 What's the status of this? If you haven't made progress, I'd like @htadeusiak to take over.

**Comment by imiao97** [comment]
Hello Jon!Thanks for reaching out. I was about to talk to you about it since we missed the meeting yesterday. I have been having a hard time to make time for it. It is interesting to me and seems doable, but it might be better to let someone else to take over if it needs to be done sometime soon. My apologies for the inconvenience and thanks for letting me know! 
Best,I-Miao


Sent from Yahoo Mail for iPhone


On Wednesday, February 13, 2019, 4:48 PM, Jon Froehlich <notifications@github.com> wrote:


@imiao97 What's the status of this? If you haven't made progress, I'd like @htadeusiak to take over.

**Comment by jonfroehlich** [comment]
Thanks @imiao97. I'm unassigning you and assigning @htadeusiak.

**Comment by htadeusiak** [comment]
Thoughts?
![test](https://user-images.githubusercontent.com/33988444/53292586-66655c00-3779-11e9-9446-17ea6e2db65b.gif)
Resizing:
![tst](https://user-images.githubusercontent.com/33988444/53292592-71b88780-3779-11e9-9f52-a766b7a642a0.gif)

**Comment by jonfroehlich** [comment]
Love it. Woohoo! Is this frontend only or does it require some backend python code too? I imagine the former?

**Comment by htadeusiak** [comment]
Yup just some HTML, CSS, and JS. Ill make a pr right now.

**Comment by htadeusiak** [comment]
I realize there's now an error loading the images for the lightbox, I'll look into it.

<img width="1440" alt="screen shot 2019-02-23 at 7 02 00 pm" src="https://user-images.githubusercontent.com/33988444/53294527-8dce2000-379d-11e9-9049-cc2622dd71c2.png">

**Comment by jonfroehlich** [comment]
Did your fix work? What's a good place to check on the test server? I don't think many of the projects have images on the test server...

**Comment by htadeusiak** [comment]
No, I'm still getting the same result. I'll keep looking into it.

https://makeabilitylab-test.cs.washington.edu/project/thermography/
^This project has an image gallery.

**Comment by htadeusiak** [comment]
In my most recent PR (PR #787), I made changes to the 'models.py' file. I created another option in the admin interface to upload a 'high quality' photo. 

The image cropping plugin that Django uses has a set pixel width and height for photos in photo galleries(I believe it's something like 340x275). This prevents them from being able to show a high quality photo. Until the Postgres and Django issue is solved, this will have to be placed on hold.

**Comment by jonfroehlich** [comment]
Students aren't really uploading images for the project gallery so closing this for now.
